Foundation/NSJSONSerialization.swift

```diff
@@ -159,7 +159,13 @@ public class JSONSerialization : NSObject {
     /* Write JSON data into a stream. The stream should be opened and configured. The return value is the number of bytes written to the stream, or 0 on error. All other behavior of this method is the same as the dataWithJSONObject:options:error: method.
      */
     public class func writeJSONObject(_ obj: AnyObject, toStream stream: NSOutputStream, options opt: WritingOptions) throws -> Int {
-        NSUnimplemented()
+        do {
```

Comment:
I would have expected this code to just call 'try' and not 'catch', therefore propagating the error outward if it happens (both from `data(withJSONObject:options:)` and `stream.write`).